frontend/src/components/home.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
```

Comment:
This was moved to `components/home/index.jsx`; please move any relevant code to that file and delete this one.